JUnit XML to HTML transformations fail in Java 24

In Java 24, our tests are not producing FAT JUnit output in the `junit.html` directory for successful and unsuccessful tests.

The following example messages are seen in the JUnit output:

```
[2024-12-18T18:40:25.000Z]  [startAnt] /home/ci/Build/workspace/ebcTestRunner/dev/autoFVT/io.openliberty.microprofile7.internal_fat/autoFVT/src/ant/launch.xml:743: Errors while applying transformations: javax.xml.transform.TransformerException: java.lang.RuntimeException: Use of the extension element 'redirect' is not allowed when extension functions are disabled by the secure processing feature or the property 'jdk.xml.enableExtensionFunctions'. To enable extension functions, set 'jdk.xml.enableExtensionFunctions' to 'true'.
[2024-12-18T18:40:25.000Z]  [startAnt] 	at java.xml/com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:790)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at java.xml/com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:395)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.taskdefs.optional.TraXLiaison.transform(TraXLiaison.java:195)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.taskdefs.XSLTProcess.process(XSLTProcess.java:853)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.taskdefs.XSLTProcess.execute(XSLTProcess.java:389)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.taskdefs.optional.junit.AggregateTransformer.transform(AggregateTransformer.java:265)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.taskdefs.optional.junit.XMLResultAggregator.execute(XMLResultAggregator.java:158)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Task.perform(Task.java:348)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Target.execute(Target.java:392)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Target.performTasks(Target.java:413)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1399)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Project.executeTarget(Project.java:1368)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1251)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Main.runBuild(Main.java:811)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.Main.startAnt(Main.java:217)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:280)
[2024-12-18T18:40:25.000Z]  [startAnt] 	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:109)
[2024-12-18T18:40:25.001Z]  [startAnt] Caused by: java.lang.RuntimeException: Use of the extension element 'redirect' is not allowed when extension functions are disabled by the secure processing feature or the property 'jdk.xml.enableExtensionFunctions'. To enable extension functions, set 'jdk.xml.enableExtensionFunctions' to 'true'.
[2024-12-18T18:40:25.001Z]  [startAnt] 	at java.xml/com.sun.org.apache.xalan.internal.xsltc.runtime.BasisLibrary.runTimeError(BasisLibrary.java:1622)
[2024-12-18T18:40:25.001Z]  [startAnt] 	at java.xml/com.sun.org.apache.xalan.internal.xsltc.runtime.BasisLibrary.runTimeError(BasisLibrary.java:1626)
[2024-12-18T18:40:25.001Z]  [startAnt] 	at java.xml/com.sun.org.apache.xalan.internal.xsltc.runtime.BasisLibrary.unallowed_extension_elementF(BasisLibrary.java:462)
[2024-12-18T18:40:25.001Z]  [startAnt] 	at jdk.translet/die.verwandlung.junit_frames.template$dot$0()
[2024-12-18T18:40:25.001Z]  [startAnt] 	at jdk.translet/die.verwandlung.junit_frames.applyTemplates()
[2024-12-18T18:40:25.001Z]  [startAnt] 	at jdk.translet/die.verwandlung.junit_frames.applyTemplates()
[2024-12-18T18:40:25.001Z]  [startAnt] 	at jdk.translet/die.verwandlung.junit_frames.transform()
[2024-12-18T18:40:25.001Z]  [startAnt] 	at java.xml/com.sun.org.apache.xalan.internal.xsltc.runtime.AbstractTranslet.transform(AbstractTranslet.java:627)
[2024-12-18T18:40:25.001Z]  [startAnt] 	at java.xml/com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:783)
[2024-12-18T18:40:25.001Z]  [startAnt] 	... 21 more
```

--- a/dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml
+++ b/dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml
@@ -725,6 +725,10 @@
 			<arg value="-Dbasedir=${basedir}" />
 			<arg value="-Ddir.log.html=${dir.log.html}" />
 			<arg value="-Ddir.log.xml=${dir.log.xml}" />
+			<!-- In Java 24, the default for 'jdk.xml.enableExtensionFunctions' was changed to false
+				 This broke our JUnit XML to HTML transformations for FAT output in junit.html
+				 Forcing value back to true - https://bugs.openjdk.org/browse/JDK-8343024 -->
+			<jvmarg line="-Djdk.xml.enableExtensionFunctions=true" />
 		</java>
 
 		<iff>

